### DISCUSSION TITLE
各位中午好

### DISCUSSION THREAD

**Discussion by canyusuo** [share121/inter-knot#4727] [崩坏：星穹铁道]
<img width="1280" alt="Honkai  Star Rail Screenshot 2024 09 10 - 10 31 48 71" src="https://github.com/user-attachments/assets/a9db6c52-f178-43bd-b191-083412fba5f8">

**Comment by canyusuo** [comment]
保底出了个鸭鸭，20抽又出了三个<img src="https://bbs-static.miyoushe.com/static/2024/05/10/5eb4431537387b22233ac3c6da58189f_4285892181442425038.png" alt="知更鸟 唱歌" width="75" >

**Comment by mztxy229** [comment]
你不许玩崩铁

**Comment by lllliufeng** [comment]
你不准玩崩铁<img src="https://bbs-static.miyoushe.com/static/2024/03/28/31a28f284bf9be6e38915547144477d7_949521742665742194.png" alt="黄泉 流泪" width="75" >

**Comment by Andrea0126** [comment]
你不好😭😭😭

**Comment by yskg487** [comment]
你不能玩崩铁！
